vyper/types/types.py

```diff
@@ -260,18 +266,27 @@ def parse_unit(item, custom_units):
         raise InvalidTypeException("Invalid unit expression", item)
 
 
+def mkstruct(name, location, members, custom_units, custom_structs):
```

Comment:
Also please just rename this to `make_struct`.

Reply:
Fixed in cececb103fc13db3181999a944719a40b17982e0